media-gfx/openscad/openscad-2019.05-r4.ebuild

```diff
@@ -46,7 +46,7 @@ RDEPEND="
 	media-libs/freetype
 	>=media-libs/glew-2.0.0:0=
 	media-libs/harfbuzz:=
-	media-libs/lib3mf
+	<media-libs/lib3mf-2
```

Comment:
Will the update cause runtime issues? Ie should this be revbumped for this...

Reply:
2019.05 will not build with lib3mf-2. Lib3mf-2 has been released after 2019.05 and support for it has not been backported

Reply:
The idea is, people who have 2019.05-r4 installed now will update to lib3mf-2, because the restriction won't hit their package database before a rebuild. So, if lib3mf-2 will cause **runtime** issues to openscad-2019.05, this ebuilds needs t obe revbumped so that won't happen.

Reply:
As the package won't build with lib3mf-2 I suspect it will cause runtime issues, whenever someone tries to im-/export a 3mf format file.
I'm gonna revbump the package then.